dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/CCLicenseStep.java

```diff
@@ -70,8 +70,8 @@
     protected static final Message T_license = 
         message("xmlui.Submission.submit.CCLicenseStep.license");
         protected static final Message T_submit_remove = message("xmlui.Submission.submit.CCLicenseStep.submit_remove");
-        protected static final Message T_no_license    = message("xmlui.Submission.submit.CCLicenseStep.no_license");
-        protected static final Message T_select_change = message("xmlui.Submission.submit.CCLicenseStep.select_change");
+        protected static final Message T_no_license    = message("webui.Submission.submit.CCLicenseStep.no_license","xmlui.Submission.submit.CCLicenseStep.no_license");
+        protected static final Message T_select_change = message("webui.Submission.submit.CCLicenseStep.select_change","xmlui.Submission.submit.CCLicenseStep.select_change");
```

Comment:
I changed the optionId not the i18n key, to avoid server side to double check for xmlui and jspui for this optionId. In any case in the PR for 6.x I follow your suggestion.